x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -535,7 +533,7 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
         isUsingFilter={isUsingFilter}
         setEnrollmentFlyoutState={setEnrollmentFlyoutState}
         clearFilters={clearFilters}
-        isCurrentRequestIncremented={isCurrentRequestIncremented}
+        queryHasChanged={queryHasChanged}
```

Comment:
@criamico I had some trouble to figure what was the goal of that and, it seems it's to show "loading agents" when loading data from view agents instead of "no agents" right?

Reply:
I don't remember after all this time 😓 
I checked [my old PR](https://github.com/elastic/kibana/pull/152555/files#diff-58111b5232d1427610e6ee73affef50f2ca7d71f2822558c4b28a711019793edR405) and it seems that `currentRequestRef?.current === 1` was just a way to show `loading agents`. 

This page has been there for long time so it has many leftovers, thanks for refactoring it.